sdk/monitor/Azure.Monitor.OpenTelemetry/README.md

```diff
@@ -1,19 +1,53 @@
 # Azure Monitor Distro client library for .NET
 
+The Azure Monitor Distro is a client library that sends telemetry data to Azure Monitor following the OpenTelemetry Specification. This library can be used to instrument your ASP.NET Core applications to collect and send telemetry data to Azure Monitor for analysis and monitoring.
+
 ## Getting started
```

Comment:
```suggestion
## Get started
```

Reply:
Azure SDK does not allow us to alter the headings, CI will throw an error.